docs/_tutorials/cache_api.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: tutorials
+permalink: /tutorials/cache-api/
+title: Cache API
+---
+
+Jekyll includes a caching API, which is used both internally as well as exposed
+for plugins, which can be used to cache the output of deterministic functions to
```

Comment:
What functions?

Reply:
How could I word this better? I am thinking like a programmer, and can’t think with words.

“You have a plugin. Parts of that plugin meet the requirements of a ‘function.’ Caching can speed up those parts of your plugin.”

Reply:
Examples are always a safe bet.